moda-contracts/script/DeployOpenReleasesFactory.s.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.21;
+
+import {Script, console2} from "forge-std/Script.sol";
+import {OpenReleasesFactory} from "../src/OpenReleasesFactory.sol";
+import {Registry} from "../src/Registry.sol";
+import {OpenReleases} from "../src/OpenReleases.sol";
+import {Upgrades} from "openzeppelin-foundry-upgrades/Upgrades.sol";
+import {IBeacon} from "@openzeppelin/contracts/proxy/beacon/IBeacon.sol";
+import {DeployedContracts} from "./utils/DeployedContracts.sol";
+
+contract DeployOpenReleasesFactory is Script {
+    function run() public {
+        uint256 privateKey = vm.envUint("DEPLOYER_PRIVATE_KEY");
+        address deployer = vm.addr(privateKey);
+        vm.startBroadcast(privateKey);
+
+        Registry registry = Registry(DeployedContracts.get("DeployRegistry.s.sol", block.chainid));
+        IBeacon beacon = IBeacon(Upgrades.deployBeacon("OpenReleases.sol", deployer));
+        OpenReleasesFactory factory = new OpenReleasesFactory(registry, address(beacon));
```

Comment:
The two releases factories don't use the Beacon pattern, you need to be passing in the implementation address here instead of `address(beacon) `